README.md

```diff
@@ -410,6 +412,20 @@ _Default: 10_
 
 Length of random salt used when generating password hash.
 
+#### __Provisioning.SetHostName__
+
+_Type: Boolean_
+_Default: y_
+
+Control if WAAgent should attempt to set the VMs hostname
+
+#### __Provisioning.UserAccounts
+
+_Type: Boolean_
+_Default: y_
+
+Control if WAAgent should configure user accounts
```

Comment:
Same as above. Maybe add an example usage when you wouldn't want to configure user accounts.

Nit: period at the end of the sentence for consistency.